src/cascadia/TerminalApp/App.cpp

```diff
@@ -270,6 +273,43 @@ namespace winrt::TerminalApp::implementation
         _newTabButton.Flyout(newTabFlyout);
     }
 
+    // Method Description:
+    // - Builds the taskbar jumplist and populates it with the user's profiles. Clicking
+    //   the jumplist items will open a new tab associated with the clicked profile.
+    // Arguments:
+    // - None
+    // Return Value:
+    // - None
+    IAsyncAction App::_CreateJumplist()
+    {
+        // It doesn't seem unreasonable to assume that Terminal will only
+        // run on platforms that support jump lists, but always better safe
+        // than sorry
+        if (JumpList::IsSupported())
+        {
+            auto jumplist = co_await JumpList::LoadCurrentAsync();
+            jumplist.SystemGroupKind(JumpListSystemGroupKind::None);
+            jumplist.Items().Clear();
+
+            for (int profileIndex = 0; profileIndex < _settings->GetProfiles().size(); profileIndex++)

```

Comment:
since we don't need the index, let's use a range-based for. We may need to capture a copy of the vector anyway.

```
for(const auto& profile: _settings->GetProfiles()) {
}
```